lua/octo/util.lua

```diff
@@ -143,4 +144,9 @@ function M.update_reactions_at_cursor(bufnr, cursor, reactions, reaction_line)
   api.nvim_buf_set_var(bufnr, "reaction_line", reaction_line)
 end
 
+function M.format_date(to_format)
+  local time_bias = date():getbias() * -1
+  return date(to_format):addminutes(time_bias):fmt("%Y %b %d %I:%M %p %Z")
```

Comment:
make format configurable with that format as the default value? we can use `vim.g.octo_date_format` for it